testplan/web_ui/testing/src/AssertionPane/AssertionTypes/GraphAssertions/graphUtils.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Helper functions used to add customisability and styling to the graph and chart assertions.
+ *
+ */
+
+export function returnStyle(graph_type){
```

Comment:
Would be good to have a one-line comment describing each function just above its definition.